guide/src/main/asciidoc/problems/badRequest.adoc

```diff
@@ -33,24 +40,40 @@ Content-Type: application/problem+json
 {
   "type": "urn:problem-type:belgif:badRequest",
   "href": "https://www.belgif.be/specification/rest/api-guide/problems/badRequest.html",
-  "status": 400,
   "title": "Bad Request",
+  "status": 400,
   "detail": "The input message is incorrect",
+  "instance": "urn:uuid:123456-1234-1235-4567489798",
   "issues": [
```

Comment:
Probably not that important, but in practice a combination of schemaValidation and non-schemaValidation issues within the same Bad Request problem is very unlikely, as the schemaValidation typically is done on an API Gateway, and the other validations are done on application level.

Reply:
yeah, it's just an example to illustrate that it could be done. 
There are cases that I know of where all types of validation or done by the backend (using generated classes with Jakarta Bean Validation). Clients shouldn't depend on order or grouping of validations.